jina/types/arrays/neural_ops.py

```diff
@@ -248,3 +247,48 @@ def visualize(
             plt.savefig(output, bbox_inches='tight', pad_inches=0.1)
         else:
             plt.show()
+
+    @property
+    def embeddings(self) -> np.ndarray:
```

Comment:
This does not belong in this mixin, I think it should belong in the GetAttrMixin

Reply:
gut GetAttrMixin is only for DocumentArray right? we want this for both.

Reply:
ok then keep it here for now